src/manifest.json

```diff
@@ -14,6 +14,15 @@
     }
   ],
   "background": { "scripts": ["background.bundle.js"], "persistent": false },
+  "commands": {
+    "toggle-meeting-mute": {
+      "suggested_key": {
+        "default": "Ctrl+Space",
+        "mac": "Ctrl+Space"
```

Comment:
Can you change this to `"MacCtrl+Space"` ? On Mac cmd-space opens spotlight search.